src/tensor_parallel/factory.py

```diff
@@ -2,6 +2,7 @@
 from typing import Optional, Sequence
 
 import torch
+import torch.distributed
```

Comment:
why: torch.distributed may not be loaded in some cases. if it isnt, the local usage of torch.distributed would fail